### DISCUSSION TITLE
Override items' title in bagpart editor

### DISCUSSION THREAD

**Discussion by intellimedhu** [OrchardCMS/OrchardCore#7947] [Q&A]
I'd like to render the red text instead of content type. Is it possible somehow?
![image](https://user-images.githubusercontent.com/65296389/101923060-3649b000-3bcf-11eb-8995-95b60578472c.png)

EventDate - date field
Category - text field
Quantity - numeric field

**Comment by deanmarcussen** [comment]
The text rendered there is an accumulation of the `ContentType` and the items `DisplayText`.

To change the display text (obvs it is empty, as there is nothing there), add the `TitlePart` to the bag item, and use a generated title, to render the dates etc

**Reply by intellimedhu** [reply]
I'll test it soon

**Comment by asimeonov** [comment]
I had a similar case lately where I needed a bit more complex title that the one displayed by default. Orchard Core offers this in a really simple way. 

- On the Content type that you created for the Bad add a Title part as @deanmarcussen suggested.
- Then you go and Edit the part
- For my case I had a linked item with content picker (StoreContentItem in my case) then I had Price and Quantity similar to your case.

This is the Custom Title that I'm using:
![image](https://user-images.githubusercontent.com/1816685/102723547-56851780-4311-11eb-95db-9ea6a2a4a5a9.png)

Pattern:
```
{% assign assigned_stores = ContentItem.Content.StoreProduct.Store.ContentItemIds | content_item_id %}
{% for assigned_store in assigned_stores %}
{{ assigned_store | display_text | append: " Q:" | append: ContentItem.Content.StoreProduct.Qantity.Value | append: " P:" | append: ContentItem.Content.StoreProduct.Price.Value}}
{% endfor %}
```
This is the one that I have generated. You can create also custom Summary template with Liquid. For doing this you need to enable ```Templates``` feature under ```Development``` category.
